www/css/_guides.scss

```diff
@@ -0,0 +1,121 @@
+.language-diff .deleted-sign,
+.language-diff .inserted-sign .inserted {
+  user-select: none;
+}
+
+.token.inserted {
+  color: #b5cea8;
+}
+
+.token.deleted {
+  color: #ce9178;
+}
+
+.markdown-body img.screenshot {
+  box-shadow: none;
+  border: none;
+  border-radius: 0;
+}
+
+.markdown-body embed {
+  background-color: #fff;
+  box-sizing: content-box;
+  max-width: 100%;
+  width: 100%;
+}
+
+.frame {
+  background-color: #0a5e9d;
+  padding: 34px 17px 10px 17px;
+  border-radius: 10px;
+  position: relative;
+  margin: 1rem 0;
+
+  &::before {
+    content: url(/img/guides/react/buttons.svg);
```

Comment:
not sure I follow, what are these / where are they used?

Reply:
Just ignore this, I didn't mean to commit this file, it's compiled not part of this PR haha